tripy/examples/diffusion/example.py

```diff
@@ -0,0 +1,222 @@
+import argparse, tempfile
+from tqdm import tqdm
+from pathlib import Path
+from PIL import Image
+import time
+
+import torch
+import cupy as cp
+import numpy as np
+
+from model import ClipTokenizer, StableDiffusion
+from weight_loader import load_from_diffusers
+import tripy as tp
+
+def tripy_diffusion(args):
+    model = StableDiffusion()
+    load_from_diffusers(model, tp.float32, debug=True)
+
+    run_start_time = time.perf_counter()
+
+    # Run through CLIP to get context
+    tokenizer = ClipTokenizer()
+    prompt = tp.Tensor([tokenizer.encode(args.prompt)])
+    print(f"Got tokenized prompt.")
+    unconditional_prompt = tp.Tensor([tokenizer.encode("")])
+    print(f"Got unconditional tokenized prompt.")
+
+    print("Compiling CLIP model...")
```

Comment:
@akhilg-nv we should have model compilation (for all the models in diffusion) be separate from execution. Then we can also introduce serialization of compiled models to disk so that can you just run standalone inference. Also, it makes the code easier to understand.